tools/spell_tagger/spell_tagger.py

```diff
@@ -13,7 +14,8 @@
 if (not languageMatch):
   raise Exception('Invalid language "{}"'.format(language))
 
-with open("spell_tagger.toml", "rb") as f:
+dir_path = os.path.dirname(os.path.realpath(__file__))
+with open(dir_path + '/spell_tagger.toml', 'rb') as f:
```

Comment:
This change allows calling the script from a different directory. It's more of a cleanup since I did the same with the `spell_extractor.py` and has nothing else to do with this change.